### ISSUE TITLE
Instruction versioning consistency and PBI Desktop on ALH VM image (was: Design a Report in Power BI Desktop, Part 1 - Minor UI change)

### ISSUE THREAD

**Issue by VitalyMCT** [MicrosoftLearning/PL-300-Microsoft-Power-BI-Data-Analyst#91]
Regarding

> In the visual fields pane (not the Fields pane—the visual fields pane is located beneath the Visualizations pane), notice that the fields are assigned to the X-axis and Column y-axis wells/areas.

The names for the wells for this visual are different in the PBI Desktop release deployed on the ALH VMs. Please update instructions accordingly.

**Comment by bpmoring** [comment]
Hi Vitaly, we are testing implementing an automatic download for the lab files. While testing this feature, they inadvertently reverted to an older version. We have updated the version of the app to June 2022. This should fix the above issue. Thanks.

**Comment by VitalyMCT** [comment]
@bpmoring Sorry, can you please elaborate on the app update, as that's a different topic from this ticket? Are you saying the official ALH images are being switched from the February 2022 release to a June 2022 release of Power BI Desktop?

**Comment by bpmoring** [comment]
Yes, previously we had updated the version of Power BI. Which is why the instructions had changed. But while testing this new feature they inadvertently reverted to an older image that led the instructions not to match the Power BI version
.

**Comment by VitalyMCT** [comment]
I've just checked both Skillable and Go Deploy ALHs. Both are on the February 2022 release. Are you saying the June 2022 upgrade is on the way from Microsoft to all the ALHs? If so, is there a timeline on that push?

Also, were the instructions regression tested on the June 2022 release? Aside from a minor update to wording on June 24th, the earlier updates precede the June 14th release date of PBI Desktop June 2022 release (please see screenshot below). I would've expected at least some breaking changes and thus a need to update the instructions. Yet there are no such commits in this repo. This to me indicates a high risk of instructions breaking in places, once the June 2022 PBI Desktop release makes it to ALHs.

The bottom line is that the instructions and the software on ALHs must be in sync, otherwise the student experience would be compromised. Please let us know your thoughts.

![image](https://user-images.githubusercontent.com/1765732/178060003-838464c9-c637-47f4-b3cb-ea0310e87b67.png)

**Comment by bpmoring** [comment]
The previous version that was loaded onto the ALHs was the May 2022 release. All instructions were updated to match that version. Because of the testing of this new feature, the version of the release was reverted to an older version (February 2022). I updated the version of Power BI on my profile, and I am submitting to Skillable to deploy across all VMs today. You will not see these changes until next week.

**Comment by VitalyMCT** [comment]
Thanks for the additional info. Here are my follow-up questions:

1) What's the plan around VM update on ALHs other than Skillable (ex. GoDeploy)? I happen to have a class running on GoDeploy in 1 week.

2) What about regression testing instructions from May 2022 release to June 2022 release? I believe from your clarification that May 2022 might've been the release the current instructions were updated for.

**Comment by bpmoring** [comment]
No problem. 
1. I will ask my lab contact what needs to be done to make sure GoDeploy is updated. I will update here when I find out. 
2. I will be taking next week to walk through the labs for the June 2022 release and make any necessary updates to the instructions. Unfortunately, I could not update to the May 2022 version which would have matched the instructions.

**Comment by bpmoring** [comment]
Ok, just confirmed the updated version of the VM will be pushed out to all ALHs. Next week will be spent focusing on any updates to instructions that are needed. Thank again, have a great weekend.

**Comment by VitalyMCT** [comment]
Thanks for the update. What's the timeline on the following item?

> the updated version of the VM will be pushed out to all ALHs

**Comment by bpmoring** [comment]
The updated version will be available on 7/15. Thanks!

**Comment by VitalyMCT** [comment]
@bpmoring GoDeploy advised me this morning that they still have not received the new VM image. Can you please advise?

**Comment by bpmoring** [comment]
It appears that due to an offsite last week, Skillable failed to deploy the updated VM to all ALHs. This has been given high priority by the team and they will send me an update when it is completed. Thanks.

**Comment by VitalyMCT** [comment]
> It appears that due to an offsite last week, Skillable failed to deploy the updated VM to all ALHs. This has been given high priority by the team and they will send me an update when it is completed. Thanks.

Is there a new timeline on the deployment? I suppose for now we need to ask the students to manually update to the May (or is it June ... see below) release.

> I will be taking next week to walk through the labs for the June 2022 release and make any necessary updates to the instructions. Unfortunately, I could not update to the May 2022 version which would have matched the instructions.

Can you please advise whether this was completed? There have been zero new commits. I take it either there were zero regressions or this activity has been delayed too.

**Comment by bpmoring** [comment]
No update from the team on whether the update has been sent or not. No new commits due to no large noticeable changes.

**Comment by bpmoring** [comment]
Just received confirmation from Skillable that the VM update has been provided to the ALHs via the LDC.

**Comment by VitalyMCT** [comment]
@bpmoring  I am looking at Skillable itself now. The VM still has the February 2022 PBI Desktop release. Please advise.

**Comment by bpmoring** [comment]
Updates are usually made on Friday due to fewer people accessing the VM. The changes should be reflected Monday. Thanks.

**Comment by VitalyMCT** [comment]
@bpmoring From Skillable:

_It has been confirmed with the Content Owner that the July 2022 version is correct. They are currently working on updating the instructions to fix any inconsistencies._

Can you please advise when the instructions here will be tested & updated for that release?
